enterprise/frontend/src/metabase-enterprise/auth/components/SettingsJWTForm/SettingsJWTForm.unit.spec.tsx

```diff
@@ -23,7 +24,7 @@ const elements = [
     description: "Is JWT authentication configured and enabled?",
     originalValue: null,
     display_name: "JWT Authentication",
-    type: "boolean",
+    type: "boolean" as InputSettingType,
```

Comment:
I think we can replace `as InputSettingType` here and below to `as const`. The reason is that we can cast invalid string like "boooolean" to `InputSettingType` and there will not be a type error. But if we use `as const` then it won't be possible to have an invalid string without errors

Reply:
yes, good catch